### ISSUE TITLE
Pixiv Ugoira/Animations

### ISSUE THREAD

**Issue by Biitap** [brave/browser-laptop#15318]
### Description

Ugoira/Animations in pixiv do not work with brave browser. 

### Steps to Reproduce

Just look at any ugoira in pixiv.net

### What version of Brave are you using?

0.56.15

**Comment by darkwiz666** [comment]
Pretty sure your "issue" is that scripts are blocked on the site, since Ugoria use HTML5 and Javascript. Disabling them allows for them to run just fine.

**Comment by Biitap** [comment]
I have set all scripts to be unblocked on both the shield and noscript extension. It is the first thing I did for websites I usually visit.  Shield is set to "All scripts allowed" and noscript is disabled.

**Comment by bsclifton** [comment]
This is an issue with the new version of Brave. Would you be able to create a new issue (making sure to include more details, on how we can reproduce the problem)?

The new issue log is located here:
https://github.com/brave/brave-browser
